app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java

```diff
@@ -1214,7 +1276,10 @@ private void assertCorrectDays(){
     private void logInTestUser() {
         _solo.enterText(_solo.getEditText("Email"), _testUser.getEmail());
         _solo.enterText(_solo.getEditText("Password"), _testUser.getPassword());
+       //  _solo.enterText(_solo.getEditText("Email"), "julie@gmail.com");
+        // _solo.enterText(_solo.getEditText("Password"), "password");
```

Comment:
This should probably be removed
- [ ] remove commented lines

Reply:
Any updates on this?

Reply:
Just running UI tests

Reply:
Can we remove these lines before we merge this into `develop`?